src/main/kotlin/mb/coronium/model/eclipse/Feature.kt

```diff
@@ -62,18 +66,15 @@ data class Feature(
         when(subNode.nodeName) {
           "plugin" -> {
             val depId = subNode.attributes.getNamedItem("id")?.nodeValue
-            val depVersionStr = subNode.attributes.getNamedItem("version")?.nodeValue
-            if(depId == null || depVersionStr == null) {
-              log.warning("Plugin dependency of $file has no id or version; skipping dependency")
+
+            if (depId == null) {
+              log.warning("Skipping plugin without id in feature.xml")
             } else {
-              val depVersion = BundleVersion.parse(depVersionStr)
-              if(depVersion == null) {
-                log.warning("Could not parse dependency version '$depVersionStr' of $file; skipping dependency")
-              } else {
-                val coordinates = Dependency.Coordinates(depId, depVersion)
-                val unpack = subNode.attributes.getNamedItem("unpack")?.nodeValue?.toBoolean() ?: false
-                dependencies.add(Dependency(coordinates, unpack))
-              }
+              val depVersionStr = subNode.attributes.getNamedItem("version")?.nodeValue
+              val unpack = subNode.attributes.getNamedItem("unpack")?.nodeValue?.toBoolean() ?: false
+              val depVersion = parseVersionStr(depVersionStr, log)
+              val coordinates = Dependency.Coordinates(depId, depVersion)
+              dependencies.add(Dependency(coordinates, unpack))
```

Comment:
I've made this a bit more flexible. It's sufficient if a `<plugin>` in feature.xml contains an ID. We can leave the version unspecified, in which case it should come from Gradle.

Reply:
The idea of these model classes is that the regular class, in this case `Feature` is always fully complete and correct (i.e., has all required values filled in and the values are correct). The builder class may be incomplete. It would be cleaner if the builder also had a builder for `Dependency`/`Coordinates`, so that the original one does not need a nullable annotation. But I think this is ok for now.